python/prophet/tests/test_diagnostics.py

```diff
@@ -236,11 +236,13 @@ def test_performance_metrics(self, ts_short, backend):
         assert set(df_horizon.columns) == {"coverage", "mse", "horizon"}
         # Skip MAPE
         df_cv.loc[0, "y"] = 0.0
```

Comment:
You'll still want to keep these calls:

```
df_horizon = diagnostics.performance_metrics(
            df_cv,
            metrics=["coverage", "mape"],
        )
assert set(df_horizon.columns) == {"coverage", "horizon"}
```

Reply:
Ah thanks, missed that! Added back in now.